core/src/test/java/org/apache/brooklyn/core/test/qa/performance/EntityPerformanceLongevityTest.java

```diff
@@ -21,7 +21,7 @@
 import org.testng.annotations.Test;
 
 
-@Test(groups={"Integration", "Acceptance"})
+@Test(groups={"Acceptance", "Performance"})
```

Comment:
If a test is part of two groups it will never execute (that's how `Broken` disables tests). Obviously it's been like this before, but better choose only one of the groups.